meta-resin-common/recipes-connectivity/resin-net-config/resin-net-config/resin-net-config

```diff
@@ -16,8 +16,10 @@ else
 fi
 
 CONFIG_NM=${CONFIG_PATH%/*}/system-connections/
-cp -r "$CONFIG_NM" /etc/NetworkManager/
-chmod 600 /etc/NetworkManager/system-connections/*
+if [ -d "$CONFIG_NM" ]; then
+	cp -r "$CONFIG_NM" /etc/NetworkManager/
+	chmod 600 /etc/NetworkManager/system-connections/*
+fi
```

Comment:
Out of curiosity, when will no system-connections directory exist?

Reply:
@floion when running ResinOS as a container - the boot partition, will only contain the config as the network stack is provided by the host.